test-data/unit/check-overloading.test

```diff
@@ -787,7 +787,11 @@ class D(A):
     def f(self, x: str) -> str: return ''
 [out]
 tmp/foo.pyi:12: error: Signature of "f" incompatible with supertype "A"
+tmp/foo.pyi:12: note: Signature of "f" in superclass: "Overload(def f(self, x: builtins.int) -> builtins.int, def f(self, x: builtins.str) -> builtins.str)"
```

Comment:
It would be nicer to have a multi-line note if you have an overloaded function, and output it like this or something:

```
...: note: Signature of "f" in superclass:
...: note:     @overload
...: note:     def f(...) -> ...
...: note:     @overload
...: note:     def f(...) -> ...
```